gnucash.modules

```diff
@@ -147,7 +147,7 @@
 
```

Comment:
libchipcard should be 221/libchipcard-5.1.4rc1.tar.gz

Reply:
yes, I changed the commit accordingly.